inst/shiny/DiagnosticsExplorer/R/Results.R

```diff
@@ -564,94 +407,91 @@ getCovariateValueResult <- function(dataSource = .GlobalEnv,
       dplyr::arrange(.data$cohortId, .data$databaseId, .data$covariateId)
   }
   if ('missingMeansZero' %in% colnames(data)) {
-    data <- data %>% 
+    data <- data %>%
       dplyr::mutate(mean = dplyr::if_else(is.na(.data$mean) &
-                                                    !is.na(.data$missingMeansZero) &
-                                                    .data$missingMeansZero  == 'Y',
+                                            !is.na(.data$missingMeansZero) &
+                                            .data$missingMeansZero == 'Y',
                                           0,
-                                          .data$mean)) %>% 
+                                          .data$mean)) %>%
       dplyr::select(-.data$missingMeansZero)
   }
   return(data)
 }
 
-getConceptDetails <- function(dataSource = .GlobalEnv,
+getConceptDetails <- function(dataSource,
                               conceptIds) {
   # Perform error checks for input variables
   errorMessage <- checkmate::makeAssertCollection()
   checkmate::assertIntegerish(x = conceptIds,
-                              min.len = 1, 
+                              min.len = 1,
                               null.ok = TRUE,
                               add = errorMessage)
   checkmate::reportAssertions(collection = errorMessage)
-  if (is(dataSource, "environment")) {
-    data <- get("concept", envir = dataSource) %>% 
-      dplyr::filter(.data$conceptId %in% conceptIds)
-  } else {
-    sql <- "SELECT *
+
+  sql <- "SELECT *
             FROM @vocabulary_database_schema.concept
             WHERE concept_id IN (@concept_ids);"
-    data <- renderTranslateQuerySql(connection = dataSource$connection,
-                                    sql = sql,
-                                    vocabulary_database_schema = dataSource$vocabularyDatabaseSchema,
-                                    concept_ids = conceptIds, 
-                                    snakeCaseToCamelCase = TRUE) %>% 
-      tidyr::tibble()
-  }
+  data <- renderTranslateQuerySql(connection = dataSource$connection,
+                                  sql = sql,
+                                  vocabulary_database_schema = dataSource$vocabularyDatabaseSchema,
+                                  concept_ids = conceptIds,
+                                  snakeCaseToCamelCase = TRUE) %>%
+    tidyr::tibble()
+
   return(data)
 }
 
-resolveMappedConceptSetFromVocabularyDatabaseSchema <- function(dataSource = .GlobalEnv, 
+resolveMappedConceptSetFromVocabularyDatabaseSchema <- function(dataSource,
                                                                 conceptSets,
                                                                 vocabularyDatabaseSchema = 'vocabulary') {
-  if (is(dataSource, "environment")) {
-    stop("Cannot resolve concept sets without a database connection")
+  if (dbms == "sqlite") {
+    stop("Cannot resolve concept sets without a full vocabulary")
```

Comment:
@azimov  this is incorrect. The cohort diagnostics standard output will have vocabulary tables. The concepts in the vocabulary tables will be just enough to resolve the concept set expression (Based on how it was resolved at the time when cohort diagnostics initially ran)

reason why this not supported for premerged file was because we did not convert the sql logic to resolve concept-set-expression to R-dply syntax.

Reply:
so now we can use that sql - because the sql is in ohdsi sql and sqlite can use OHDSI sql with translation!